app/js/components/manageApps/ManageApps.jsx

```diff
@@ -167,7 +168,7 @@ export default class ManageApps extends React.Component {
         });
         installedAddons = installedOwas.concat(installedModules);
         return Promise
-          .all(response.data.results.map(data => axios.get(`${ApiHelper.getAddonUrl()}?modulePackage=${data.packageName}`)
+          .all(response.data.results.map(data => axios.get(`${ApiHelper.getAddonUrl()}/${data.packageName}`)
```

Comment:
This change reverses the effect of `AOM-113`. Is there a reason for that?

Reply:
@segunolalive the previous URL request causes the request for updates to return no updates found. The issue it tries to resolve is less critical and can be adjusted with a UX/UI implementation adjustment

Reply:
The URL i used for retrieving the developer's name in `AOM-103` has somewhat broken the `check for update link` and created other bugs in the app. So like @Elbertbiggs360 suggested, this PR takes priority for now and we will try to fix resolve `AOM-103` using a UX/UI implementation

Reply:
That's okay.

Reply:
So can you reopen that ticket and give the author these details? As in tell them why you have reversed their change?

Reply:
Alright, thanks